### PR TITLE
SubGroups filled, sources updated

### PR DESCRIPTION
**Pull request by StasReichert**
The contents of the languages.tsv and the sources.bib were changed. This time I was asked for a Google API key in the process although skipping it didn't seem to destroy anything.

### CODE REVIEW

File: cldf/cldf-metadata.json [hunk 1]
```diff
@@ -16,19 +16,19 @@
...
             "rdf:about": "https://github.com/tupian-language-resources/tuled",
-            "dc:created": "v0.10-120-g0c546c7"
+            "dc:created": "v0.10-122-g58fbf6f"
         },
         {
             "rdf:type": "prov:Entity",
             "dc:title": "Glottolog",
             "rdf:about": "https://github.com/glottolog/glottolog",
-            "dc:created": "v4.3"
+            "dc:created": "v4.2.1-13-g7970be034b"
```
Comment (LinguList) [inline cldf/cldf-metadata.json:25]: can you please re-run and add the keyword `--glottolog-version=v4.3`? The entry here suggests you run this without a specific version, and your glottolog repository seems to be outdated. Same holds for concepticon (please git-pull and use `--concepticon-version=v2.4.0`

**Review by LinguList — CHANGES_REQUESTED** [review]

File: .zenodo.json [hunk 2]
```diff
@@ -33,6 +33,9 @@
         }
     ],
     "communities": [
+        {
+            "identifier": "lexibank"
+        },
```
Comment (xrotwang) [inline .zenodo.json:38]: @StasReichert you seem to be running an older version of pylexibank - which had this issue of adding redundant communities. Could you update (`pip install -U pylexibank`)?

File: cldf/languages.csv [hunk 3]
```diff
@@ -1,80 +1,80 @@
...
-Akuntsu,Akuntsu,akun1241,Akuntsu,aqz,South America,-12.8322,-60.9716,Tupian,Tuparí,
-Amanaye,Amanaye,aman1266,Amanayé,ama,South America,-3.8,-48.66,Tupian,Tupi-Guarani,
-Amondawa,Amondawa,amun1246,Amundava,adw,South America,-9.04023,-62.3064,Tupian,Tupi-Guarani,
-Anambe,Anambe,anam1249,Anambé,aan,South America,-2.71119,-49.303,Tupian,Tupi-Guarani,
-AnambeEhrenreich,Anambe Ehrenreich,anam1251,Anambe of Ehrenreich,,South America,-4.5,-50.5,Tupian,Tupi-Guarani,
+Akuntsu,Akuntsu,akun1241,,,,-12.8322,-60.9716,,Tuparí,
+Amanaye,Amanaye,aman1266,,,,-3.8,-48.66,,Tupi-Guarani,
+Amondawa,Amondawa,amun1246,,,,-9.04023,-62.3064,,Tupi-Guarani,
+Anambe,Anambe,anam1249,,,,-2.71119,-49.303,,Tupi-Guarani,
+AnambeEhrenreich,Anambe Ehrenreich,anam1251,,,,-4.5,-50.5,,Tupi-Guarani,
```
Comment (xrotwang) [inline cldf/languages.csv:6]: This regression is an effect of not downloading the language data from the google sheet. Did you get an error message with instructions how to create a google API key to use as key file?

Comment (StasReichert) [reply cldf/languages.csv]: I did not get any instructions with the error message, just a prompt for a Google API key file. I was able to run the cldfbench with glottolog version 4.3 and concepticon version 2.4.0 afterwards. I also updated pylexibank as you asked before running the code. Should I push the results or is there something I should do first?

Comment (xrotwang) [reply cldf/languages.csv]: I already updated the files in this PR. So no need to do anything now.

**Pull request merged**